ticketauth/src/main/kotlin/dk/ufst/ticketauth/automated/SelectUserActivity.kt

```diff
@@ -0,0 +1,96 @@
+package dk.ufst.ticketauth.automated
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Intent
+import android.content.res.Resources
+import android.graphics.Color
+import android.os.Bundle
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup.LayoutParams
+import android.view.ViewGroup.LayoutParams.MATCH_PARENT
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.Button
+import android.widget.LinearLayout
+import android.widget.ScrollView
+import android.widget.TextView
+import androidx.core.view.setPadding
+import dk.ufst.ticketauth.BuildConfig
+import org.json.JSONArray
+
+class SelectUserActivity : Activity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setResult(RESULT_CANCELED)
+        intent?.extras?.getString("users")?.let {
+            val usersJson = JSONArray(it)
+            setContentView(buildContentView(usersJson))
+        } ?: run {
+            finish()
+        }
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun buildContentView(users: JSONArray): View {
+        val bgColor = Color.parseColor("#14143C")
+        val textColor = Color.parseColor("#FFFFFF")
+        val ll = LinearLayout(this)
+        ll.orientation = LinearLayout.VERTICAL
+        ll.layoutParams = LayoutParams(MATCH_PARENT, MATCH_PARENT)
+        ll.setPadding(0, toDp(16f).toInt(), 0, 0)
+        ll.setBackgroundColor(bgColor)
+        TextView(this)
```

Comment:
Cool that you have kept it simple here and not used Compose.

Reply:
Thanks but it dawned upon that I AM including appcompat (for activity result launchers) and here I use the regular framework. I don't see any problem with this simple usage, but might be nicer to use appcompat in a future update. .im gonna make stuff like colors configurable anyway as you requested in the demo :)